_sass/_lists.scss

```diff
@@ -0,0 +1,33 @@
+.list--bullets {
+  margin-left: 0;
+
+  @media only screen and (min-width: $breakpoint-medium) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li {
+    padding-left: 25px;
+    position: relative;
+    list-style: none;
+    box-sizing: border-box;
+
+    @media only screen and (min-width: $breakpoint-medium) {
+      width: 50%;
+    }
+
+    &:before {
+      content: '';
+      background-image: url("http://assets.ubuntu.com/sites/ubuntu/latest/u/img/patterns/tick-orange.svg");
```

Comment:
Can you use the new asset server here please: https://assets.ubuntu.com/v1/c4b02d61-tick-orange.svg